diff --git a/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js b/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js
index 34c674f..e875618 100755
--- a/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js
+++ b/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js
@@ -31,7 +31,7 @@ class PasswordInputText extends React.Component {
     };
 
     render() {
-        const { iconSize, iconColor, label, style } = this.props;
+        const { iconSize, iconColor, label, style, iconStyle } = this.props;
         const { icEye, isPassword } = this.state;
 
         return (
@@ -41,7 +41,7 @@ class PasswordInputText extends React.Component {
                     ref={this.getRef}
                     secureTextEntry={isPassword}
                     label={label} />
-                <Icon style={styles.icon}
+                <Icon style={[styles.icon, iconStyle]}
                     name={icEye}
                     size={iconSize}
                     color={iconColor}
